testsuite/src/it/java/io/quarkus/ts/startstop/CodeQuarkusTest.java

```diff
@@ -68,18 +80,26 @@
     public static final List<List<CodeQuarkusExtensions>> supportedEx = CodeQuarkusExtensions.partition(4, CodeQuarkusExtensions.Flag.SUPPORTED);
     public static final List<List<CodeQuarkusExtensions>> notSupportedEx = CodeQuarkusExtensions.partition(1, CodeQuarkusExtensions.Flag.NOT_SUPPORTED);
     public static final List<List<CodeQuarkusExtensions>> mixedEx = CodeQuarkusExtensions.partition(1, CodeQuarkusExtensions.Flag.MIXED);
+    
+    public static final Stream<CodeQuarkusExtensions> supportedExWithCodeStarter() {
+        return Arrays.asList(CodeQuarkusExtensions.QUARKUS_CONFIG_YAML,
+                CodeQuarkusExtensions.QUARKUS_LOGGING_JSON,
+                CodeQuarkusExtensions.QUARKUS_RESTEASY,
+                CodeQuarkusExtensions.QUARKUS_RESTEASY_JACKSON,
+                CodeQuarkusExtensions.QUARKUS_SPRING_WEB).stream();
```

Comment:
Let me know if you prefer loading the extensions dynamically or adding more extensions like Quarkus Qute.

Reply:
Add `quarkus-qute`, it's TP and that should be covered too